stacker/blueprints/redis.py

```diff
@@ -0,0 +1,107 @@
+from troposphere import Ref, ec2, Output, GetAtt
+from troposphere.elasticache import ReplicationGroup, SubnetGroup
+
+from .base import Blueprint
+
+SUBNET_GROUP = 'RedisSubnetGroup'
```

Comment:
I've been ditching prefixes like this lately - instead of `RedisSubnetGroup` I just use `SubnetGroup`.  Not sure there is much point to prefixing them.